profiles/schema.py

```diff
@@ -373,27 +374,53 @@ def __resolve_reference(self, info, **kwargs):
 
 
 class EmailInput(graphene.InputObjectType):
-    id = graphene.ID()
-    email = graphene.String(description="Email address.")
-    email_type = AllowedEmailType(description="Email address type.", required=True)
     primary = graphene.Boolean(description="Is this primary mail address.")
 
 
+class CreateEmailInput(EmailInput):
+    email = graphene.String(description="Email address.", required=True)
+    email_type = AllowedEmailType(description="Email address type.", required=True)
+
+
+class UpdateEmailInput(EmailInput):
+    id = graphene.ID(required=True)
+    email = graphene.String(description="Email address.")
+    email_type = AllowedEmailType(description="Email address type.")
+
+
 class PhoneInput(graphene.InputObjectType):
-    id = graphene.ID()
+    primary = graphene.Boolean(description="Is this primary phone number.")
+
+
+class CreatePhoneInput(PhoneInput):
     phone = graphene.String(description="Phone number.", required=True)
     phone_type = AllowedPhoneType(description="Phone number type.", required=True)
-    primary = graphene.Boolean(description="Is this primary phone number.")
+
+
+class UpdatePhoneInput(PhoneInput):
+    id = graphene.ID(required=True)
+    phone = graphene.String(description="Phone number.")
+    phone_type = AllowedPhoneType(description="Phone number type.")
 
 
 class AddressInput(graphene.InputObjectType):
-    id = graphene.ID()
+    country_code = graphene.String(description="Country code")
```

Comment:
Should this description end with a period for consistency?

Reply:
Though, on the other hand, I think most descriptions might look better without a period, since they aren't proper sentences.

Reply:
🤔 Could we file a ticket to make the descriptions & stuff consistent? There are still some work to do (look for `TODO`s 😄) with the API documentation so this is something we could squeeze in there.